tests/UnitTests/CoreGraphics.drawing/DrawingTest.cpp

```diff
@@ -34,10 +38,14 @@ void testing::DrawTest<TComparator>::SetUp() {
     CGSize size = CanvasSize();
 
     auto deviceColorSpace = woc::MakeStrongCF<CGColorSpaceRef>(CGColorSpaceCreateDeviceRGB());
-    _context.reset(CGBitmapContextCreate(
-        nullptr, size.width, size.height, 8, size.width * 4, deviceColorSpace, kCGImageAlphaPremultipliedFirst));
+    _context.reset(
+        CGBitmapContextCreate(nullptr, size.width, size.height, 8, size.width * 4, deviceColorSpace, kCGImageAlphaPremultipliedFirst));
     ASSERT_NE(nullptr, _context);
 
+#if WINOBJC // Validate that the results are correct even under batched drawing from _CGContextPushBegin/PopEndDraw
```

Comment:
Agreed, I'm not sure what you're envisioning for how we know if we control the data (will eventually call Create/GetImage). Leaving as-is for now.

---
In reply to: [98318678](https://github.com/Microsoft/WinObjC/pull/1870#discussion_r98318678) [](ancestors = 98318678)